### PR TITLE
[2201.2.x] Add missing code actions to object fields

### LINKED ISSUE #37614: [Bug]: Missing code actions in object fields
### Description

Consider the source below where change variable type code action and type cast code action are not suggested for object fields.
1. Missing change variable type code action
Screenshot 2022-09-02 at 14 20 33

2. Missing add type cast code action
Screenshot 2022-09-02 at 14 28 24

### Steps to Reproduce

```
class Person2 {

    private int age = 22.2;

}
```

### Affected Version(s)

2201.2.0 (Swan Lake Update 2)

### OS, DB, other environment details and versions

_No response_

### Labels

LanguageServer

### Related issue(s) (optional)

_No response_

### Suggested label(s) (optional)

_No response_

### Suggested assignee(s) (optional)

_No response_

### PR DESCRIPTION
**Pull request by LakshanWeerasinghe**
## Purpose
In the object fields numeric typecast code action and change variable code action were missing. This pr will add these missing code actions to object fields.

Fixes #37614

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [x] Added necessary tests
  - [x] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [x] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/CodeActionContextTypeResolver.java [hunk 1]
```diff
@@ -79,6 +82,18 @@ public Optional<TypeSymbol> transform(LetVariableDeclarationNode letVariableDecl
         return getTypeDescriptorOfVariable(letVariableDeclarationNode);
     }
 
+    @Override
+    public Optional<TypeSymbol> transform(ObjectFieldNode objectFieldNode) {
+        return objectFieldNode.typeName().apply(this);
+    }
+
+    @Override
+    public Optional<TypeSymbol> transform(BuiltinSimpleNameReferenceNode builtinSimpleNameReferenceNode) {
```
Comment (malinthar) [inline language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/CodeActionContextTypeResolver.java:91]: Can't we use SimpleNameReferenceNode instead? And could you provide an example which is covered by this transform method

**Review by malinthar — CHANGES_REQUESTED** [review]
Changes looks good! Let's address the following.

File: language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/CodeActionContextTypeResolver.java (hunk 1 above)
Comment (LakshanWeerasinghe) [reply language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/CodeActionContextTypeResolver.java:91]:
```
class A {

float f1 = 10.2;
decimal d1 = 1.2;

float f2 = f1 <cursor> / d1;

}
```
Above scenario is covered by this case.

File: language-server/modules/langserver-core/src/test/resources/codeaction/change-var-type/source/change_var_type_in_object_fields.bal [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class TestClass {
+        
+    string f1 = 12.2;
+    
+    int a = "string";
+    
+    string arr = ["a", "b"];
```
Comment (IMS94) [inline language-server/modules/langserver-core/src/test/resources/codeaction/change-var-type/source/change_var_type_in_object_fields.bal:7]: Shall we update this test case to have a qualifier like `private` (`private string arr = xxx`)?

**Review by IMS94 — CHANGES_REQUESTED** [review]

**Comment by LakshanWeerasinghe** [comment]
Master PR : #37969

**Review by IMS94 — APPROVED** [review]

**Comment by IMS94** [comment]
@LakshanWeerasinghe shall we send this to master instead?

**Comment by IMS94** [comment]
Just noticed that we have merged this to master already. Since this is not an urgent fix, not sending this to 2201.2.x branch.

**Pull request closed without merging**